src/psse_export.jl

```diff
@@ -937,6 +935,8 @@ function write_to_buffers!(
     bus_number_mapping = md["bus_number_mapping"]
     transformer_name_mapping = md["transformer_name_mapping"]
 
+    _value_or_default(val, default) = isnothing(val) ? default : val
```

Comment:
Define this helper function near the top of the file, with the other helper functions, not within another function